jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/Jar.java

```diff
@@ -84,7 +84,9 @@ public Integer call() {
         return 1;
       }
 
-      Containerizer containerizer = Containerizers.from(commonCliOptions, logger);
+      CacheDirectories cacheDirectories =
+          CacheDirectories.from(commonCliOptions, jarFile.toAbsolutePath().getParent());
```

Comment:
I don't think this will work. @mpeddada1 can you check what `jarFile.toAbsolutePath()` evaluates to when running `jib jar app.jar` (i.e., `jarFile` is simply `app.jar`).

Reply:
Yup. Calling `jarFile.toAbsolutePath()` with `jib jar sample.jar` results in `/Users/<username>/<path to jib folder>/sample.jar`. So it resolves it for the current directory.

Reply:
Oh, I was wrong for the whole time about `Path.toAbsolutePath()`. I realized its [Javadoc](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#toAbsolutePath()) says it actually resolves the path against a file system.

> Otherwise, this method resolves the path in an implementation dependent manner, typically by resolving the path against a file system default directory. 

> `/Users/<username>/<path to jib folder>/sample.jar`

You mean `.../<parent path of sample.jar>/sample.jar` in the end, right? For example, if the current directory is `/foo`, and if I run `foo $ jib bar/sample.jar`, then `jarFile.toAbsoluatePath()`, will correctly resolve to `/foo/bar/sample.jar` (as long as the file exists), right?

Reply:
> You mean `.../<parent path of sample.jar>/sample.jar` in the end, right?
Yes.

> You mean `.../<parent path of sample.jar>/sample.jar` in the end, right? For example, if the current directory is `/foo`, and if I run `foo $ jib bar/sample.jar`, then `jarFile.toAbsoluatePath()`, will correctly resolve to `/foo/bar/sample.jar` (as long as the file exists), right?

Or it would be `/Users/username/foo/bar/sample.jar` for unix and something like `C:\Users\username\foo\bar\sample.jar` for windows?

Reply:
^ well depending on the filesystem default directory according to the doc

Reply:
Okay, you assumed the current working directory was `/Users/username/foo`. In my comment above, the working directory I assumed is `/foo`. So in my case, it will resolve to `/foo/bar/sample.jar`.

In any case, looks like `jarFile.toAbsoluatePath()` does the right thing that we want, so I guess no issue here.